Setup/Product.wxs

```diff
@@ -363,16 +363,16 @@
         <File Source="$(var.ArtifactsPublishPath)\Plugins\GitExtensions.Plugins.Bitbucket.dll" />
       </Component>
       <Component Id="TeamCityIntegration.dll" Guid="*">
-        <File Source="$(var.ArtifactsPublishPath)\Plugins\TeamCityIntegration.dll" />
+        <File Source="$(var.ArtifactsPublishPath)\Plugins\GitExtensions.TeamCityIntegration.dll" />
```

Comment:
These changes should no longer be needed

Reply:
`dotnet publish` fails without them

Reply:
This will fail the CI build.
Are you trying to publish a debug build?

Here's a build with this change reverted:
```
PS D:\Development\gitextensions> dotnet build .\scripts\native.proj -c Release --verbosity q --nologo /bl:.\artifacts\log\native.binlog
C:\Program Files\dotnet\sdk\6.0.403\MSBuild.dll -nologo -consoleloggerparameters:Summary -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Program Files\dotnet\sdk\6.0.403\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Program Files\dotnet\sdk\6.0.403\dotnet.dll -maxcpucount -property:Configuration=Release -restore -verbosity:m -verbosity:q /bl:.\artifacts\log\native.binlog .\scripts\native.proj

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:17.53
PS D:\Development\gitextensions> dotnet build -c Release --verbosity q --nologo /bl:.\artifacts\log\build.binlog /p:ContinuousIntegrationBuild=true
C:\Program Files\dotnet\sdk\6.0.403\MSBuild.dll -nologo --property:ContinuousIntegrationBuild=true -property:Configuration=Release -consoleloggerparameters:Summary -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Program Files\dotnet\sdk\6.0.403\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Program Files\dotnet\sdk\6.0.403\dotnet.dll -maxcpucount -restore -verbosity:m -verbosity:q /bl:.\artifacts\log\build.binlog .\GitExtensions.sln

Build succeeded.    0 Warning(s)    0 Error(s)

Time Elapsed 00:00:48.79
PS D:\Development\gitextensions> dotnet publish -c Release --no-build /bl:.\artifacts\log\publish.binlog /p:ContinuousIntegrationBuild=true
MSBuild version 17.3.2+561848881 for .NET
C:\Program Files\dotnet\sdk\6.0.403\MSBuild.dll --property:ContinuousIntegrationBuild=true -property:NoBuild=true -property:Configuration=Release -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Program Files\dotnet\sdk\6.0.403\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Program Files\dotnet\sdk\6.0.403\dotnet.dll -maxcpucount -target:Publish -verbosity:m /bl:.\artifacts\log\publish.binlog .\GitExtensions.sln
  GitExtensions -> D:\Development\gitextensions\artifacts\Release\publish\GitExtensions\  Downloading 'GitExtensions.PluginManager.2.0.0.160.zip'.  Zipping directory "D:\Development\gitextensions\artifacts\Release\publish\GitExtensions\" to "D:\Development\gitextensions\artifacts\Release\publish\GitExtensions-Portable-33.33.33.0-34ba1be28.zip".      
  MSBuild version 17.4.0-preview-22428-01+14c24b2d3 for .NET Framework
  C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\msbuild.exe /bl:D:\Development\gitextensions\artifacts\Release\log\\setup.binlog /p:Configuration=Release;Platform=x86;WiXVersion 
  =3.11.2;Version=33.33.33.0;ArtifactsBinPath=D:\Development\gitextensions\artifacts\Release\bin\;ArtifactsPublishPath=D:\Development\gitextensions\artifacts\Release\publish\GitExtensions\;OutputPath=D:\De 
  velopment\gitextensions\artifacts\Release\publish;TargetName=GitExtensions-33.33.33.0-34ba1be28 /t:Build /v:q D:\Development\gitextensions\GitExtensions\\..\Setup\Setup.wixproj
  ICSharpCode.TextEditor -> D:\Development\gitextensions\Externals\ICSharpCode.TextEditor\Project\bin\Release\net6.0-windows\publish\
PS D:\Development\gitextensions>
```

Reply:
I would like to use Debug builds, even if the usability is decreased after Asserts are no longer shown.

Why not keep builds the same and remove condition `$(ContinuousIntegrationBuild) != true`?